Formula/daemontools.rb

```diff
@@ -22,6 +22,40 @@ def install
     end
   end
 
+  def caveats; <<~EOS
+    You must create /service directory as root before starting svscan.
+    Then you can start svscan with following.
+    `sudo brew services start daemontools`
+
+    In other words, you need to execute the commands in the following order.
+
+    1. `sudo mkdir /service`
+    2. `sudo brew services start daemontools`
```

Comment:
No need to repeat this command. Ident both these commands by two spaces and remove the "in other words"

Reply:
Thanks. I fixed at 69de5fd.